.env.example

```
@@ -1,6 +1,10 @@
 NEXT_PUBLIC_BASE_API_URL=
 NEXT_PUBLIC_WS_BASE_URL=
 NEXT_PUBLIC_API_TOKEN=
+NEXT_PUBLIC_WS_SUBSCRIBE_DEST=/user/queue/chat
+NEXT_PUBLIC_WS_SEND_DEST=/ws/chat/send
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_WS_SEND_DEST key should go before the NEXT_PUBLIC_WS_SUBSCRIBE_DEST key

(UnorderedKey)